src/Components/Aspire.Milvus.Client/Aspire.Milvus.Client.csproj

```diff
@@ -0,0 +1,20 @@
+<Project Sdk="Microsoft.NET.Sdk">
```

Comment:
Can you also update:

* https://github.com/dotnet/aspire/blob/main/src/Components/Aspire_Components_Progress.md
* https://github.com/dotnet/aspire/blob/main/src/Components/Telemetry.md